yapf/yapflib/file_resources.py

```diff
@@ -23,6 +23,27 @@
 from lib2to3.pgen2 import tokenize
 
 from yapf.yapflib import py3compat
+from yapf.yapflib import style
+
+
+LOCAL_STYLE = '.style.yapf'
```

Comment:
Github seems to have expunged the previous comments (they're not even on the commit), but IIRC one suggestion was to move this var to style.py.

...At the moment the tests fail :(